src/conference/alpescraft/Mountains/MountainRight.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {Easing, Img, interpolate, staticFile, useCurrentFrame} from 'remotion';
+
+export const MountainRight: React.FC = () => {
+	const frame = useCurrentFrame();
+
+	const moveRight = interpolate(frame, [0, 10], [-300, -20], {
+		easing: Easing.elastic(1),
+		extrapolateRight: 'clamp',
+	});
+
+	return (
+		<Img
+			src={staticFile('images/conferences/alpescraft/Mountain-right.svg')}
+			style={{
+				position: 'absolute',
+				right: moveRight,
+				bottom: 0,
+				width: '20%',
+			}}
+		/>
+	);
+};
```

Comment:
Tu as beaucoup de duplication de code avec `src/conference/alpescraft/Mountains/MountainLeft.tsx` alors que tu as juste une ligne style qui change et l'asset de l'image ? On devrait pouvoir centraliser ça non ?

Reply:
Did you mean "Why not using props" ^^'